### ISSUE TITLE
Start Screen: Buy Button?

### ISSUE THREAD

**Issue by mtg1986** [IBVSoftware/CollaBoard-Feedback#471]
At the moment I can only register or request an Eval version. Should we add the option to buy?

![image](https://user-images.githubusercontent.com/19570958/35328782-cd380db4-00fd-11e8-9f2a-77602275fdb8.png)

**Comment by gsantopaolo** [comment]
@mtg1986 agree, the buy will only open the internet site page where we have the payment system.
If you want we can add a param to the html to make site understand if request is coming from a team device or not.
Let us know.
All the rest will came later when we will change the business model..

**Comment by corradocavalli** [comment]
Added in release 2.0.12, clicking will open this web page https://ibvsolutions.com/en/collaboard-en/ let me know if a more appropriate is available.

**Comment by tamarawasser** [comment]
@corradocavalli When I click on the Button "Buy" I reach the mentioned page, that's okay. But we should set an anker on the page that when I click on the link I directly reacht the part of the page where the Prices are visible.

**Comment by mtg1986** [comment]
Please make sure that from the German Version of CollaBoard people are directed to the German page and so. For the Italian version I would link to the english website.

**Comment by corradocavalli** [comment]
Agree,
Do it and let me know anchor tag.

On 9 Feb 2018, at 11:10, Tamara Wasser <notifications@github.com<mailto:notifications@github.com>> wrote:


@corradocavalli<https://github.com/corradocavalli> When I click on the Button "Buy" I reach the mentioned page, that's okay. But we should set an anker on the page that when I click on the link I directly reacht the part of the page where the Prices are visible.

**Comment by corradocavalli** [comment]
It now point to German page when language is DE.

Da: Michael Görög<mailto:notifications@github.com>
Inviato: venerdì 9 febbraio 2018 12:21
A: IBVSoftware/CollaBoard-Feedback<mailto:CollaBoard-Feedback@noreply.github.com>
Cc: Corrado Cavalli<mailto:corrado@outlook.com>; Mention<mailto:mention@noreply.github.com>
Oggetto: Re: [IBVSoftware/CollaBoard-Feedback] Start Screen: Buy Button? (#471)


Please make sure that from the German Version of CollaBoard people are directed to the German page and so. For the Italian version I would link to the english website.

**Comment by gsantopaolo** [comment]
@tamarawasser  this needs to be tested before next release happening this week, thanks

**Comment by mtg1986** [comment]
Thanks for the button. 
To finish the task it would be great if the link goes directly to the pricing section on our page: 

For the English Link
https://ibvsolutions.com/en/collaboard-en/#pricing

For the German Link
https://ibvsolutions.com/de/collaboard/#preise

**Comment by corradocavalli** [comment]
Links updated in v 2.0.20
